src/frontend/app/features/cloud-foundry/cf-page.types.ts

```diff
@@ -4,3 +4,8 @@ import { InjectionToken, Injectable } from '@angular/core';
 export class BaseCF {
   guid: string;
 }
+
+@Injectable()
+export class BaseCFOrg {
```

Comment:
There's two different ways of fetching cf, org and space id (using these or directly from activatedRoute). Need to be consistent in method and type. Would suggest something like the following to be used everywhere
```
 @Injectable()
 export class ActiveRouteCfOrgSpace {	 
    cfGuid: string;	 
    orgGuid: string;	 
    spaceGuid: string;	 
 }
```
with
```
function getActiveRouteCfOrgSpace(activatedRoute: ActivatedRoute) {
  return {
    guid: activatedRoute.snapshot.params.orgId
... etc
  };
}
```